golem-rib/src/function_name.rs

```diff
@@ -22,6 +22,20 @@ use serde::{Deserialize, Serialize};
 use std::borrow::Cow;
 use std::fmt::Display;
 
+#[derive(Debug, PartialEq, Eq, Clone, Encode, Decode)]
+pub enum ParsedFunctionSite {
+    Global,
+    Interface {
+        name: String,
+    },
```

Comment:
To avoid spurious conflicts in a project like this, it's best to avoid moving code and definitions around within a file. If this is your IDE doing this, I recommend turning off the feature.

Reply:
I appreciate your suggestion.  This has happened to me before too, so thank you for your help!